### ISSUE TITLE
Gettext PO file validation errors (problem with plural forms)

### ISSUE THREAD

**Issue by saboter** [jdriscoll/django-photologue#94]
Hi,
before i start to mess around in my fork i want to be sure this is a real issue and not some misunderstanding on my side.

I have translated the rest of strings on transifex for Slovak language and i wanted to try it out in my app. The gettext msgfmt failed with following erros:

```
CommandError: Execution of msgfmt failed: /Path/to/sk/LC_MESSAGES/django.po:33: a format specification for argument 'gallery', as in 'msgstr[0]', doesn't exist in 'msgid_plural'
 /Path/to/sk/LC_MESSAGES/django.po:46: a format specification for argument 'gallery', as in 'msgstr[0]', doesn't exist in 'msgid_plural'
 /Path/to/sk/LC_MESSAGES/django.po:75: a format specification for argument 'photo', as in 'msgstr[0]', doesn't exist in 'msgid_plural'
 /Path/to/sk/LC_MESSAGES/django.po:88: a format specification for argument 'photo', as in 'msgstr[0]', doesn't exist in 'msgid_plural'
msgfmt: found 4 fatal errors
```

There are problems with missing argument in plural strings e.g.:

```
#: admin.py:79
#, python-format
msgid "The gallery '%(gallery)s' has been successfully added to %(site)s"
msgid_plural "The selected galleries have been successfully added to %(site)s"
msgstr[0] "Označená galéria '%(gallery)s' bola úspešne priradená k %(site)s"
msgstr[1] "Označené galérie boli úspešne priradené k %(site)s"
msgstr[2] "Označené galérie boli úspešne priradené k %(site)s"

```

The missing argument needs to be added if want to make it pass the validation e.g.:

```
#: admin.py:79
#, python-format
msgid "The gallery '%(gallery)s' has been successfully added to %(site)s"
msgid_plural "The selected galleries '%(gallery)s' have been successfully added to %(site)s"
msgstr[0] "Označená galéria '%(gallery)s' bola úspešne priradená k %(site)s"
msgstr[1] "Označené galérie '%(gallery)s' boli úspešne priradené k %(site)s"
msgstr[2] "Označené galérie '%(gallery)s' boli úspešne priradené k %(site)s"

```

Question is, if actually possible to use the combination of singular and plural msgid's with different argument placeholders.

In other words is this correct way how to use ungettext (https://github.com/jdriscoll/django-photologue/blob/master/photologue/admin.py#L81)?

```
msg = ungettext(
            "The gallery '%(gallery)s' has been successfully removed from %(site)s",
            "The selected galleries have been successfully removed from %(site)s",
            len(queryset)
        ) % {'site': current_site.name, 'gallery': queryset.first()}
```

Some examples for usage of ungettext: https://docs.djangoproject.com/en/dev/topics/i18n/translation/#pluralization.

Also found similar issue here https://github.com/transifex/transifex/issues/184.

Thanks.
